contrib/pg_tde/src/keyring/keyring_kmip_ereport.c

```diff
@@ -22,4 +22,5 @@ void kmip_ereport(bool throw_error, const char *msg, int errCode)
         elog(ereport_level, (msg));
         #pragma GCC diagnostic pop
     }
-}
\ No newline at end of file
+}
+
```

Comment:
Why is this file changed in this commit? It should probably be changed in its own commit.